mssqlcli/mssql_cli.py

```diff
@@ -489,6 +507,8 @@ def _evaluate_command(self, text):
                     click.secho("Aborted!", err=True, fg='red')
                     break
 
+            contains_secure_statement = self.has_security_statement_in_cmd(sql)
+
```

Comment:
perf nit: aren't we running the function twice? Once for file history and once here executing the command? It'd be nice todo it once, though it might not make sense if it complicates the code.

Reply:
The file history is tied to prompt tool kitwhich attempts to load the previous type statement if the first letter matches it. This code block is for preventing the smart completion from loading the statements.

Reply:
Understood. But the question still stands, aren't we parsing the same query twice?

Reply:
Yes, I can add a flag in the filehistory object that we can check if the result query contained security keywords. Let me know your thoughts.